web/satellite/src/components/project/billing/BillingArea.vue

```diff
@@ -76,16 +82,49 @@
 		}
 	}
 
-	@media screen and (max-height: 880px) {
-		.billing-overflow {
+	@media screen and (max-height: 900px) {
+		.billing-container__table-holder {
 			overflow-y: scroll;
-			height: 600px;
+			height: 555px;
+
+			&::-webkit-scrollbar, &::-webkit-scrollbar-track, &::-webkit-scrollbar-thumb {
+				visibility: hidden;
+			}
 		}
 	}
 
-	@media screen and (max-height: 700px) {
-		.billing-overflow {
-			height: 570px;
+	@media screen and (max-height: 860px) {
+		.billing-container__table-holder {
+			overflow-y: scroll;
+			height: 535px;
+		}
+	}
+
+	@media screen and (max-height: 835px) {
+		.billing-container__table-holder {
+			overflow-y: scroll;
+			height: 495px;
+		}
+	}
+
+	@media screen and (max-height: 815px) {
+		.billing-container__table-holder {
+			overflow-y: scroll;
+			height: 495px;
+		}
+	}
+
+	@media screen and (max-height: 795px) {
+		.billing-container__table-holder {
+			overflow-y: scroll;
+			height: 475px;
+		}
+	}
+
+	@media screen and (max-height: 780px) {
```

Comment:
The rules in this block is the same as the one above?

Reply:
Thanks for review! Changes applied